### ISSUE TITLE
Train caffe model failed with error code -11

### ISSUE THREAD

**Issue by qjqjqjj** [NVIDIA/DIGITS#1914]
Hi,

When I train caffe model with object detection it always throw me error code -11 or -3, does anyone know whats the error about?

Below is the error code that I have:
`Train net output #1: loss_coverage = 3.21673 (* 1 = 3.21673 loss) `
`Iteration 12586, lr = 0.000607665`
`Iteration 12617 (0.369076 iter/s, 83.9935s/31 iter), loss = 8.37578`
`Train net output #0: loss_bbox = 0.949957 (* 2 = 1.89991 loss)`
`Train net output #1: loss_coverage = 22.5043 (* 1 = 22.5043 loss)`
`Iteration 12617, lr = 0.00060692`
`Iteration 12648 (0.36866 iter/s, 84.0883s/31 iter), loss = 7.63125`
`Train net output #0: loss_bbox = 0.333911 (* 2 = 0.667823 loss)`
`Train net output #1: loss_coverage = 3.09236 (* 1 = 3.09236 loss)`
`Iteration 12648, lr = 0.000606175`
`Iteration 12679 (0.368692 iter/s, 84.081s/31 iter), loss = 9.85937`
`Train net output #0: loss_bbox = 6.00124 (* 2 = 12.0025 loss)`
`Train net output #1: loss_coverage = 8.44669 (* 1 = 8.44669 loss)`
`Iteration 12679, lr = 0.000605432`
`Snapshotting to binary proto file snapshot_iter_12700.caffemodel`
`Snapshotting solver state to binary proto file snapshot_iter_12700.solverstate`
`Iteration 12700, Testing net (#0)`
`Ignoring source layer train-data`
`Ignoring source layer train-label`
`Ignoring source layer train-transform`

Thanks!
